### PR TITLE
Add old vanish back

### PR DESCRIPTION
**Pull request by Matwarey**
You may ask.. Why bring back old vanish?

Well, some worlds is over 1GB or more or have no storage to turn on spectator mode, and import it back into their realm, plus the old vanish was better, sure, it couldn't fly though walls and stuff but it's better to have this still in the current version of scythe today + in my option it's a way better way to spy on targets who may be hacking or other things.

Otherwise, we should keep the old vanish has a option if they want to use it or not, if not, that's fine, if you want it, then it can happen if MrDiamond64 accepts this pull request. (P.S DM Me On Discord if any bugs)

### CODE REVIEW

File: functions/checks/assets/vanish.mcfunction [hunk 1]
```diff
@@ -0,0 +1,2 @@
+effect @s[tag=oldvanish] invisibility 9999 255 true
+effect @s[tag=oldvanish] night_vision 9999 255 true
```
Comment (MrDiamond64) [inline functions/checks/assets/vanish.mcfunction:2]: missing YOU ARE VANISHED text

File: scripts/commands/handler.js [hunk 2]
```diff
@@ -69,6 +70,7 @@ export function commandHandler(player, message) {
         else if (config.customcommands.ban && commandName === "ban") ban(message, args);
         else if (config.customcommands.notify && commandName === "notify") notify(message);
         else if (config.customcommands.vanish && commandName === "vanish" || commandName === "v") vanish(message);
+		else if (config.customcommands.oldvanish && commandName === "oldvanish" || commandName === "ov") vanish(message);
```
Comment (MrDiamond64) [inline scripts/commands/handler.js:73]: improve formatting here

File: scripts/data/config.js [hunk 3]
```diff
@@ -32,6 +32,7 @@ export default
         "notify": true,
         "tag": true,
         "vanish": true,
+	"oldvanish": true,
```
Comment (MrDiamond64) [inline scripts/data/config.js:35]: also fix the formatting here

**Comment by MrDiamond64** [comment]
scale changing and godmode is missing

(full list of things that need to be added)[https://github.com/MrDiamond64/Scythe-AntiCheat/commit/7c1f16dfee626689b5cd1064177d820b68293924]

File: scripts/data/config.js (hunk 3 above)
Comment (Matwarey) [reply scripts/data/config.js:35]: I noticed that and already fixed it

**Comment by MrDiamond64** [comment]
why change uuids?

**Comment by Matwarey** [comment]
I felt liked changing them, but i did fix every one of them but the top one.

**Comment by 3kh0** [comment]
Old vanish is better tbh

**Pull request closed without merging**